docs/OpenCart.md

```diff
@@ -0,0 +1,184 @@
+
+On successfully saving the BTCPay extension automatically created a webhook so that OpenCart can get notified when payments settle of fail. To double check it was successful. You can do that by editing the BTCPay extension settings again, if you see the "Webhook Data" field filled out like this:
+![BTCPay OpenCart: Save OpenCart Settings form](./img/opencart/oc3--12--webhook-success.png) 
+
+As you can see on the BTCPay extension settings you can further customize the order statuses depending on the [invoice statuses](https://docs.btcpayserver.org/Invoices/#invoice-statuses) and other common settings. The defaults should be a good starting point but feel free to adjust to your use case. 
+
+## 3. Test the checkout
+
+Everything is ready to go now. Do a small test purchase and make sure the order status gets updated according to the BTCPay invoice status. On BTCPay Server invoice details you can see if the webhook events were fired successfully.
+
+## Troubleshooting
+
+### Enable debug mode
+
+If you have an error during checkout you can enable the debugging mode on the BTCPay extension settings. Menu: Go to "Extensions -> extensions" select "Payments" on the "Choose Extension Type" dropdown and edit BTCPay Server extension.
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--20--debug-mode-enable.png) 
+
+You can now find the debug output in the error log in menu "System -> Maintenence -> Error Logs".
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--21--error-logs.png) 
+
+*Please make sure to disable it after debugging finished or otherwise it will fill up your error logs.*
+
+
+
+**Example Error**:   
+> 2022-05-24 21:10:50 ERROR Error during POST to https://btcpay.example.com/api/v1/stores/4kD5bvAF5j8DokHqAzxb6MFDV4ikabcdefghijklm/invoices. Got response (401): {&quot;code&quot;:&quot;unauthenticated&quot;,&quot;message&quot;:&quot;Authentication is required for accessing this endpoint&quot;}
+
+- This means there is some authentication error. Likely your api key does not have permission create invoices for that store. Make sure you gave the api key the right permissions and you give it to the right store and also entered that in OpenCart payment configuration form.
+
+- Another reason could be that you use a legacy api key. The legacy api keys are located in store settings -> Access Tokens. But you need to create an account api key which is located in Account -> Manage Account -> tab "API Keys". See section [2.2 Create an API key and configure permissions](#22-create-an-api-key-and-configure-permissions).
+
+
+## The order states do not update although the invoice has been paid
+Please check the details of your invoice if there were any errors on sending the webhook request. Some hosting providers, firewall setups or security extensions may block POST requests to your site which lead to a http status of "403 forbidden". 
```

Comment:
```suggestion
Please check your invoice details to see if there were any errors on sending the webhook request. Some hosting providers, firewall setups, or security extensions may block POST requests to your site, which leads to an HTTP status of "403 forbidden". 
```

Reply:
I think the , after "setups," is now too much because we have an "or" already?